testutil/compose/compose/main.go

```diff
@@ -106,6 +107,17 @@ func newAutoCmd() *cobra.Command {
 	return cmd
 }
 
+func newBuildLocalCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "build-local",
+		Short: "Builds the obolnetwork/charon:local docker container from the local source code. Note this requires the CHARON_REPO env var.",
```

Comment:
`--charon-repo` flag? not necessary though

Reply:
No, env var is much better UX